pkgs/development/python-modules/azure-mgmt-relay/default.nix

```diff
@@ -23,7 +23,7 @@ buildPythonPackage rec {
   build-system = [ setuptools ];
 
   dependencies = [
-    msrestazure
+    msrest
     azure-common
     azure-mgmt-core
     azure-mgmt-nspkg
```

Comment:
No longer needed, see https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-relay_1.1.0/sdk/relay/azure-mgmt-relay/setup.py#L86